src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+// import Big from './big.mjs';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit
+          {' '}
+          <code>src/App.js</code>
+          {' '}
+          and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
```

Comment:
Let's do so some clearing 🙂  try to remove all unnecessary code or file;
![image](https://user-images.githubusercontent.com/31897434/92557006-24204480-f263-11ea-83b8-bb7a5a7ff6db.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/


You could just `h1` the name of the app for now.

Reply:
Done.
Thanks for your suggestion bellom! :).